extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsInfoItemExtractor.java

```diff
@@ -224,10 +237,20 @@ public String getUploaderName() throws ParsingException {
     @Override
     public String getUploaderUrl() throws ParsingException {
         try {
-            return "https://www.youtube.com/channel/" + JsonUtils.getString(json,
+            return "https://www.youtube.com/channel/" + JsonUtils.getString(getCommentRenderer(),
                     "authorEndpoint.browseEndpoint.browseId");
         } catch (final Exception e) {
             return EMPTY_STRING;
         }
     }
+
+    @Override
+    public Page getReplies() throws ParsingException {
+        try {
+            final String id = JsonUtils.getString(JsonUtils.getArray(json, "replies.commentRepliesRenderer.contents").getObject(0), "continuationItemRenderer.continuationEndpoint.continuationCommand.token");
+            return new Page(url, id);
+        } catch (final Exception e) {
+            return null;
```

Comment:
I guess if `YoutubeCommentsInfoItemExtractor` is used as a reply extractor this `return null` would kick in for YouTube, wouldn't it? Add a comment to explain this.